### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by elhamatokhi**
- In this pull request I created a new branch.
- I added media queries to make my website responsive.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,6 +15,11 @@
...
       <nav class="icon">
         <img src="Images/Union.png" alt="Menu-icon" />
+        <ul>
+          <li><a href="#" >Contact</a></li>
+          <li><a href="#" >About</a></li>
+          <li><a href="#" >Portfolio</a></li>
+        </ul>   
       </nav>
     </header>
     <div class="mainContainer">
```
Comment (SajjadAhmad14) [inline index.html]: The decoration images for the headline are not positioned correctly. Kindly see the design and place them in the right position.

Your design:
yours-head

Original design:

head-o

File: index.html [hunk 2]
```diff
@@ -155,32 +160,32 @@ <h2 class="title">Tonic</h2>
         <!--End card section 1-->
 
         <!--card section 1-->
-        <div class="card card1">
```
Comment (SajjadAhmad14) [inline index.html:158]: The work section has broken images for all the projects on both mobile and desktop views. The title should have `#091E42` color. Moreover please use the hover effect for the `See Project` button.
picture

File: style.css [hunk 3]
```diff
@@ -443,3 +457,136 @@ button:hover {
...
+    gap: 30px;
+    padding: 24px;
+  }
+
+  .card--content .title {
+    align-self: flex-start;
+    font-size: 40px;
+  }
+
+  #AboutmySelf {
```
Comment (SajjadAhmad14) [inline style.css:553]: The color for the `About` section title should be `#091E42` and the color for the `Get My Resume` button text should be `#396DF2`. The `Get My Resume` button should have a hover effect as defined in the design. Moreover, `Languages` should have icons as can be seen in the design.


Your design:
your-languages


Original design:
languages

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Elhama, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 3 above)
Comment (elhamatokhi) [reply style.css:553]: Thanks for your kind review.
I have added some functionality, I think we just need to click on it.

**Review by algerina — APPROVED** [review]
Hi @elhamatokhi ,
 
Good job so far!

May I highlight the following:

- The repo looks professional. :heavy_check_mark: 
- There are no linter errors :heavy_check_mark: 
- You have descriptive PR :heavy_check_mark: 
- Your project looks awesome :heavy_check_mark: 
- You have implemented all the changes required by the previews TSE :heavy_check_mark: 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
  


 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**